### PR TITLE
 [FIX #988] Disable drag and drop

### LINKED ISSUE #988: Disable drag and drop of files into the app
## The Issue
Based on the discussion in https://github.com/lbryio/lbry-app/pull/549#issuecomment-362819852, we should disable file drag & drop into the app until it can support publishing, otherwise, it causes confusion because the file is simply shown in the app. 

### Steps to reproduce
1. Drag a picture or video file into the LBRY app
2. LBRY displays the video/image
3. No way to exit, need to close the app and reopen (completely or from tray)

### Expected behaviour
Prompt publish feature or do nothing

### Actual behaviour
Opens the file

## System Configuration

 

- LBRY Daemon version: 0.18.2
- LBRY App version: 0.20.0
- LBRY Installation ID:
- Operating system: Windows

### PR DESCRIPTION
**Pull request by ykris45**
## Preview

https://spee.ch/a/ScreenCapture2018-2-26-042956.mp4

### CODE REVIEW

File: src/renderer/index.js [hunk 1]
```diff
@@ -62,6 +62,18 @@ ipcRenderer.on('window-is-focused', () => {
   dock.setBadge('');
 });
 
+'use strict';
```
Comment (seanyesmunt) [inline src/renderer/index.js]: This isn't needed

Comment (ykris45) [reply src/renderer/index.js]: on line 65  guys

File: src/renderer/index.js [hunk 2]
```diff
@@ -62,6 +62,18 @@ ipcRenderer.on('window-is-focused', () => {
...
+
+document.addEventListener('dragover', function (event) {
+  event.preventDefault();
+  return false;
+}, false);
```
Comment (seanyesmunt) [inline src/renderer/index.js]: You have some extra stuff that isn't needed.
```
document.addEventListener('dragover', event => {
  event.preventDefault();
})
```

Comment (ykris45) [reply src/renderer/index.js]: i remove that and test on my local setup

File: CHANGELOG.md [hunk 3]
```diff
@@ -38,7 +38,7 @@ Web UI version numbers should always match the corresponding version of LBRY App
  * Added auto-update ([#808](https://github.com/lbryio/lbry-app/pull/808))
 
 
-### Changed
+### Change
```
Comment (seanyesmunt) [inline CHANGELOG.md]: This needs to stay `"Changed"`

Comment (ykris45) [reply CHANGELOG.md]: okay

**Pull request closed without merging**